pynicotine/headless/application.py

```diff
@@ -73,8 +74,30 @@ def on_confirm_quit(self, _only_on_active_uploads):
         cli.prompt(_("Do you really want to exit? %s") % responses, callback=self.on_confirm_quit_response)
 
     def on_invalid_password(self):
-        log.add(_("User %s already exists, and the password you entered is invalid. Please choose another username "
-                  "if this is your first time logging in."), config.sections["server"]["login"])
+
+        log.add(_("User %(user)s already exists, and the password you entered is invalid. Type %(command)s to "
+                  "log in with another username or password."), {
```

Comment:
Split this string, because console word wrap obscures the hint, and the first sentence is reusable in the gui.

`_("User %(user)s already exists, and the password you entered is invalid.")`
`+ "\n" + _("Type %(command)s to log in with another username or password.")`